### ISSUE TITLE
pip install lapjv doesn't work, but compiling from source does.

### ISSUE THREAD

**Issue by geometrikal** [src-d/lapjv#2]
I tried to install lapjv using `pip/pip3 install lapjv` on both Windows and Mac and got errors. 

Downloading the source files and compiling / installing via `setup.py` works fine. Any ideas?

Windows error installing via pip:
```
C:\WINDOWS\system32>pip install lapjv
Collecting lapjv
  Using cached lapjv-1.3.0.tar.gz
Requirement already satisfied: numpy in c:\users\chaos\anaconda3\lib\site-packages (from lapjv)
Building wheels for collected packages: lapjv
  Running setup.py bdist_wheel for lapjv ... error
  Complete output from command C:\Users\chaos\Anaconda3\python.exe -u -c "import setuptools, tokenize;__file__='C:\\Users\\chaos\\AppData\\Local\\Temp\\pip-build-q8o85_ei\\lapjv\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" bdist_wheel -d C:\Users\chaos\AppData\Local\Temp\tmp1orybszxpip-wheel- --python-tag cp35:
  running bdist_wheel
  running build
  running build_ext
  building 'lapjv' extension
  creating build
  creating build\temp.win-amd64-3.5
  creating build\temp.win-amd64-3.5\Release
  C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\x86_amd64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD -IC:\Users\chaos\Anaconda3\lib\site-packages\numpy\core\include -IC:\Users\chaos\Anaconda3\include -IC:\Users\chaos\Anaconda3\include "-IC:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE" "-IC:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\ATLMFC\INCLUDE" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.15063.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\NETFXSDK\4.6.1\include\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.14393.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.14393.0\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.14393.0\winrt" /EHsc /Tppython.cc /Fobuild\temp.win-amd64-3.5\Release\python.obj -fopenmp -std=c++11 -march=native -ftree-vectorize
  cl : Command line warning D9002 : ignoring unknown option '-fopenmp'
  cl : Command line warning D9002 : ignoring unknown option '-std=c++11'
  cl : Command line warning D9002 : ignoring unknown option '-march=native'
  cl : Command line warning D9002 : ignoring unknown option '-ftree-vectorize'
  python.cc
  c:\users\chaos\appdata\local\temp\pip-build-q8o85_ei\lapjv\lap.h(7): error C2988: unrecognizable template declaration/definition
  c:\users\chaos\appdata\local\temp\pip-build-q8o85_ei\lapjv\lap.h(7): error C2059: syntax error: '('
  c:\users\chaos\appdata\local\temp\pip-build-q8o85_ei\lapjv\lap.h(215): error C2143: syntax error: missing ';' before '{'
  c:\users\chaos\appdata\local\temp\pip-build-q8o85_ei\lapjv\lap.h(215): error C2447: '{': missing function header (old-style formal list?)
  python.cc(16): error C2065: 'py_lapjv': undeclared identifier
  python.cc(102): warning C4244: 'initializing': conversion from 'npy_intp' to 'int', possible loss of data
  python.cc(123): error C3861: 'lap': identifier not found
  python.cc(130): error C3861: 'lap': identifier not found
  error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\BIN\\x86_amd64\\cl.exe' failed with exit status 2
```

Building from source repo:
```
running build
running build_ext
building 'lapjv' extension
creating build
creating build\temp.win-amd64-3.5
creating build\temp.win-amd64-3.5\Release
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\x86_amd64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD -IC:\Users\chaos\Anaconda3\lib\site-packages\numpy\core\include -IC:\Users\chaos\Anaconda3\include -IC:\Users\chaos\Anaconda3\include "-IC:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE" "-IC:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\ATLMFC\INCLUDE" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.15063.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\NETFXSDK\4.6.1\include\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.14393.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.14393.0\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.14393.0\winrt" /EHsc /Tppython.cc /Fobuild\temp.win-amd64-3.5\Release\python.obj /openmp /std:c++latest /arch:AVX2
python.cc
python.cc(102): warning C4244: 'initializing': conversion from 'npy_intp' to 'int', possible loss of data
python.cc(124): warning C4800: 'int': forcing value to bool 'true' or 'false' (performance warning)
python.cc(131): warning C4800: 'int': forcing value to bool 'true' or 'false' (performance warning)
creating C:\Users\chaos\Downloads\lapjv-master\lapjv-master\build\lib.win-amd64-3.5
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\x86_amd64\link.exe /nologo /INCREMENTAL:NO /LTCG /DLL /MANIFEST:EMBED,ID=2 /MANIFESTUAC:NO /LIBPATH:C:\Users\chaos\Anaconda3\libs /LIBPATH:C:\Users\chaos\Anaconda3\PCbuild\amd64 "/LIBPATH:C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\LIB\amd64" "/LIBPATH:C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\ATLMFC\LIB\amd64" "/LIBPATH:C:\Program Files (x86)\Windows Kits\10\lib\10.0.15063.0\ucrt\x64" "/LIBPATH:C:\Program Files (x86)\Windows Kits\NETFXSDK\4.6.1\lib\um\x64" "/LIBPATH:C:\Program Files (x86)\Windows Kits\10\lib\10.0.14393.0\um\x64" /EXPORT:PyInit_lapjv build\temp.win-amd64-3.5\Release\python.obj /OUT:build\lib.win-amd64-3.5\lapjv.cp35-win_amd64.pyd /IMPLIB:build\temp.win-amd64-3.5\Release\lapjv.cp35-win_amd64.lib
python.obj : warning LNK4197: export 'PyInit_lapjv' specified multiple times; using first specification
   Creating library build\temp.win-amd64-3.5\Release\lapjv.cp35-win_amd64.lib and object build\temp.win-amd64-3.5\Release\lapjv.cp35-win_amd64.exp
Generating code
Finished generating code
```

**Comment by vmarkovtsev** [comment]
Thanks for noticing, I did not update the package on pypi.

**Comment by vmarkovtsev** [comment]
Fixed by v1.3.1
